net-p2p/retroshare/retroshare-0.6.7.ebuild

```diff
@@ -5,7 +5,7 @@ EAPI=7
 
 inherit flag-o-matic desktop qmake-utils xdg
 
-DESCRIPTION="P2P private sharing application"
+DESCRIPTION="F2F, private and secure file sharing application"
```

Comment:
now that you edit it I would change it to this "Friend to friend, private and secure communication application"

F2F is used commonly for other things other then friend to friend
and RetroShare offer much more then file sharing

Disclaimer: upstream developer here

Reply:
I know that retroshare offers much more but did not know how to describe it.
Do you think that just "sharing" is good enough?

Reply:
maybe communication and sharing? Btw sharing only seems better then file sharing

Reply:
seems good now